migrations/20200905210049-init.js

```diff
@@ -1310,6 +1310,14 @@ module.exports = {
           type: 'LONGTEXT',
           allowNull: true
         },
+        chatBridgeDCPrefix: {
```

Comment:
Good that you thought of upgrading the migrations! However, for existing instances, this migration has already ran. So when people update, this will not run.

Instead, you should make a new migration that adds these columns to make sure that existing installs also get these new columns

See the file `migrations/20210810081812-player-last-death.js`, this one does pretty much the same thing as this PR requires.